### ISSUE TITLE
eslint --init gets stuck

### ISSUE THREAD

**Issue by 1234filip** [eslint/eslint#10403]
**Tell us about your environment**

* **OS** Windows 10 64-bit
* **Shell** Default Windows CMD
* **ESLint Version:** v4.19.1
* **Node Version:** v10.2.0  
* **npm Version:** v6.1.0

**What parser (default, Babel-ESLint, etc.) are you using?** default probably, idk what a parser is

**Please show your full configuration:** none

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
eslint --init
? How would you like to configure ESLint? Answer questions about your style
```

**What did you expect to happen?** That I'm going to get questions

**What actually happened? Please include the actual, raw output from ESLint.** No actual input prompt or questions. I can't type anything and Ctrl+C doesn't work. I have to force quit the terminal

**Comment by platinumazure** [comment]
Hi @1234filip, thanks for the issue.

I'm confused about your ESLint version. We haven't released anything 10.x in ESLint. Could you please run `eslint --version` and update the issue with the result? Thanks!

**Comment by 1234filip** [comment]
Sorry i mixed up the eslint and node version

**Comment by platinumazure** [comment]
Thanks! Could you tell us what operating system and shell/command prompt you are using?

**Comment by 1234filip** [comment]
Updated

**Comment by SIkebe** [comment]
Same here.
* OS Windows 7 64-bit
* Shell Default Windows CMD
* ESLint Version: v4.19.1
* Node Version: v10.3.0
* npm Version: v6.1.0

**Comment by g-plane** [comment]
Possibly useful information:
- Works on Linux (haven't tested on macOS, maybe OK), **doesn't work on Windows**
- Works on Node.js 8 and Node.js 9, **doesn't work on Node.js 10**
- **Doesn't work on CMD, PowerShell, MSYS (with Hyper Terminal)**

**Comment by Aladdin-ADD** [comment]
maybe fixed by #10422

**Comment by Aladdin-ADD** [comment]
Is this still a issue to you? `5.0.0-rc.0` has been released.😄  @1234filip
